static/dogma.go

```diff
@@ -0,0 +1,44 @@
+package static
+
+import (
+	"go/types"
+
+	"golang.org/x/tools/go/ssa"
+)
+
+const (
+	// dogmaPkgPath is the full path of dogma package.
+	dogmaPkgPath = "github.com/dogmatiq/dogma"
+)
+
+// dogmaPackage encapsulates information about the dogma package and the types
```

Comment:
I definitely like this separate `dogmaPackage` type to encapsulate its data. Also `lookupDogmaPackage()` is quite neat too.